### PR TITLE
Tweaks to dotnet-helix-machines lifecycle process 

### PR DESCRIPTION
**Pull request by MattGal**
### To double check:

* [x] The right tests are in and and the right validation has happened.  Guidance:  https://github.com/dotnet/arcade/tree/main/Documentation/Validation _(doc only)_

### CODE REVIEW

File: Documentation/Operations/Helix-Machine-Management/Helix-End-Of-Life-Process.md [hunk 1]
```diff
@@ -104,29 +104,56 @@ If you feel this removal is in error, or believe a specific expiration should be
...
+- Review [the day's main pipeline executions](https://dnceng.visualstudio.com/internal/_build?definitionId=596&_a=summary&repositoryFilter=3&branchFilter=152037)
+- If any warnings about EOL queues arise (see guidance for specific types below):
+  - Check whether this removal represents the "last" of this operating system (no other queues have this OS). If so, get confirmation from the Release PM team or confirmation from “Matrix of Truth” to ensure its removal is acceptable and mention this in issues.
+    - If removal is deemed inappropriate, make pull requests to the dotnet-helix-machines repo extending the time to a new, agreed-upon, date.
+    - If pull requests are created, monitor subsequent builds in the pipeline until it has succeeded; 
+  - Open issues in the dotnet/arcade for all actions, with the "First Responder" tag. Add to list of queues for end-of-week update.
+
+- If the main build has failed (red): 
+  - Ping DncEng First Responders Teams Channel and ask for next steps.
```
Comment (garath) [inline Documentation/Operations/Helix-Machine-Management/Helix-End-Of-Life-Process.md:122]:
```suggestion
  - Ping [DncEng First Responders Teams Channel](https://teams.microsoft.com/l/channel/19%3aafba3d1545dd45d7b79f34c1821f6055%40thread.skype/First%2520Responders?groupId=4d73664c-9f2f-450d-82a5-c2f02756606d&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) and ask for next steps. 
```

If you want, a link to be fancy.

File: Documentation/Operations/Helix-Machine-Management/Helix-End-Of-Life-Process.md [hunk 2]
```diff
@@ -104,29 +104,56 @@ If you feel this removal is in error, or believe a specific expiration should be
...
+  - Optionally, create issue in dotnet/arcade with the "First Responders" label requesting investigation.
+
+##### Weekly (Currently, most rollouts are on Wednesdays):
+- Find the rollout pipeline.  Rollout occurs around 9 AM in the [dotnet-helix-machines-ci pipeline](https://dnceng.visualstudio.com/internal/_build?definitionId=596&_a=summary) with branch 'production'. Contact the dnceng team if this is not occurring or you lack permissions.
+- Even if there are no tasks to be done, always create an issue using [this template](https://dnceng.visualstudio.com/internal/_workitems/create/Task?templateId=7599e1ed-4c83-45cd-ad97-10ce36dbbb20&ownerId=3f024d6c-9884-4f38-a598-025fca9dfcd2) and put it into the "active" state.
+- Fill out the sections of the template:
+  - Link to completed pipeline
+  - Status of the rollout (completed normally, aborted, rolled back, etc)
+  - List of any images/queues producing warnings in the pipeline (errors will stop us from running it)
+    - Open issues in the [dotnet/arcade](https://github.com/dotnet/arcade/) and list links to them under the "queues that will expire in the next 7 days" and "after the next 7 days" headings.
```
Comment (garath) [inline Documentation/Operations/Helix-Machine-Management/Helix-End-Of-Life-Process.md:132]:
```suggestion
    - Create new issues in [dotnet/arcade](https://github.com/dotnet/arcade/) identifying the queue and linking to the build. Include links to these issues under the "queues that will expire in the next 7 days" or "after the next 7 days" headings as appropriate.
```

Sentence seems muddled to me. Suggestion is trying to tease out the steps a bit more.

File: Documentation/Operations/Helix-Machine-Management/Helix-End-Of-Life-Process.md [hunk 3]
```diff
@@ -104,29 +104,56 @@ If you feel this removal is in error, or believe a specific expiration should be
...
 
 1. Review the date and confirm using internet search / .NET Release PM Team calendar.  **Note the "Matrix of Truth" data, once it exists, supersedes any data in "Estimated Removal dates"
 2. If the date is incorrect, consult with DncEng Operations v-team (or, use judgment) to determine a new date and extend it via pull-request to this repository
 3. If the date is valid, remove the definition via pull request to this repository.  If we have erroneously never communicated this state, you may use discretion to set a date sometime in the future to keep the current status quo, but please include a comment over the definition explaining why.
 
+
+When the build to be rolled out contains warnings with `{QueueOrImageName} has estimated removal date: {Date}, in the next three weeks. Please include this in communications about upcoming rollouts.`:
+
+1. Review the date and confirm using internet search / .NET Release PM Team calendar and release notes.  Example: [.NET Core 6.0 release notes](https://github.com/dotnet/core/blob/main/release-notes/6.0/supported-os.md)
```
Comment (garath) [inline Documentation/Operations/Helix-Machine-Management/Helix-End-Of-Life-Process.md:146]: It feels like we should create an independent policy document that just states the OS and where to get EOL info. And also how the PM calendar applies. 

But that can be a change for later.

Comment (MattGal) [reply Documentation/Operations/Helix-Machine-Management/Helix-End-Of-Life-Process.md:146]: right.  This was written before the "matrix of truth" work started and I intend to update it again when that epic woohoos.

**Review by garath — APPROVED** [review]

**Pull request merged**